### DISCUSSION TITLE
Rocky Linux Profile

### DISCUSSION THREAD

**Discussion by omok314159** [ComplianceAsCode/content#6981] [Q&A] [answered]
Hi, 
I'm afraid that this is stupid question I guess, but I wish to ask in here.

When we will make Rocky Linux contents like ol7, sles15, we need to create our own several profiles.
In this case, can we copy RHEL8 profile and modify it for rocky8(I mean legal problem)?
I know this is not good as OSS developer attitude, so I guess we need to create those profiles by ourselves.
But I wish to confirm it in the beggining.
(I believe we need to create our own profiles, then I wish to tell Rocky community that we need much resource for creating/supporting).

We made this kind of directory structure last December for supporting rocky8 in ComplianceAsCode(I regret that I didn't ask in here 5 month ago), and updating it in each CaC release. Now we can create ssg-rocky8-ds.xml or others, and now we are checking is it working in anaconda.
https://github.com/atomicturtle/openscap/tree/main/ComplianceAsCode/content_for_supporting_rocky8/files/rocky8

Also I know that you already made PR for Rocky Linux as CentOS.
https://github.com/ComplianceAsCode/content/pull/6936

**Comment by omok314159** [comment]
I decided to create our own profiles. Please ignore this. Thanks.

**Comment by marcusburghardt** [comment, marked as answer]
@omok314159 , I think it would be nice if you bring a new product to the project. There are rules and benchmarks valid for many distros and we can sum the efforts to continually improve the project.

This meantime, it was created a documentation about the introduction of new products:
https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#creating-a-new-product

I also invite you and any other person interested to contribute to participate in our Gitter space: https://gitter.im/Compliance-As-Code-The/content

**Comment by ggbecker** [comment]
Rocky Linux should probably be a derivative product of RHEL the same it is with CentOS. Nowadays we can even considering keeping the profiles from RHEL.

**Reply by marcusburghardt** [reply]
@ggbecker , is there any documentation to guide the introduction of a new derivative?
